Headers/Public/VLCMediaList.h

```diff
@@ -104,12 +104,13 @@ extern NSString *const VLCMediaListItemDeleted;
 - (void)insertMedia:(VLCMedia *)media atIndex:(NSUInteger)index;
 
 /**
- * remove a media from a given position
+ * remove and return the boolean result of the operation
```

Comment:
remove media at position index and return true if the operation was successful. An unsuccessful operation occurs when the index is greater than the medialists count